std_msgs/String type
opencv_apps/Point2D center
std_msgs/Float32 zoom
std_msgs/Float32 yaw_angle
opencv_apps/Point2D[] points